Last-Update: 2013-01-27
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: correct default matplotlib backend QtAgg --> Qt4Agg
 QtAgg is no longer available.

--- a/cmd/seekwatcher
+++ b/cmd/seekwatcher
@@ -830,10 +830,10 @@
 
     parser.add_option("-i", "--interactive", help="Use matplotlib interactive",
                       action="store_true", default=False)
     parser.add_option("", "--backend",
-               help="matplotlib backend (QtAgg, TkAgg, GTKAgg) case sensitive",
-               default="QtAgg")
+               help="matplotlib backend (Qt4Agg, TkAgg, GTKAgg) case sensitive",
+               default="Qt4Agg")
     parser.add_option("-T", "--title", help="Graph Title", default="")
     parser.add_option("-R", "--reads-only", help="Graph only reads",
                       default=False, action="store_true")
     parser.add_option("-W", "--writes-only", help="Graph only writes",
